package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "yieldnest-cross-chain",
+  "description": "Contracts and scripts for deploying cross chain yieldnest tokens using LayerZero.",
+  "version": "0.1.0",
+  "devDependencies": {
+    "@layerzerolabs/lz-evm-messagelib-v2": "^2.1.27",
+    "@layerzerolabs/lz-evm-oapp-v2": "^2.1.18",
+    "@layerzerolabs/lz-evm-protocol-v2": "^2.1.27",
+    "@layerzerolabs/lz-evm-v1-0.7": "^2.3.40",
+    "@openzeppelin/contracts": "4.9.2",
+    "@openzeppelin/contracts-upgradeable": "4.9.2",
```

Comment:
we have to make sure we *Do Not* use OZ 4.x anywhere. 

we must bump both to OZ 5.x